editor/plugins/texture_region_editor_plugin.cpp

```diff
@@ -70,7 +70,7 @@ void TextureRegionEditor::_region_draw() {
 	VS::get_singleton()->canvas_item_add_set_transform(edit_draw->get_canvas_item(), Transform2D());
 
 	if (snap_mode == SNAP_GRID) {
-		Color grid_color = Color(1.0, 1.0, 1.0, 0.15);
+		Color grid_color = get_color("grid_major_color", "Editor");
```

Comment:
Wrong rebase?

Reply:
Not sure i didnt know wich line was the lastest, will change it if you want